great_expectations/util.py

```diff
@@ -1766,8 +1772,8 @@ def get_context(
     |  get_context params   |    Env Not Config'd |  Env Config'd |
     +-----------------------+---------------------+---------------+
     | ()                    | Local               | Cloud         |
-    | (ge_cloud_mode=True)  | Exception!          | Cloud         |
-    | (ge_cloud_mode=False) | Local               | Local         |
+    | (cloud_mode=True)  | Exception!          | Cloud         |
```

Comment:
align the table, please *twitch* *twitch*

Reply:
done, sorry 😭